components/build-service/staging/base/kustomization.yaml

```diff
@@ -3,3 +3,14 @@ kind: Kustomization
 resources:
```

Comment:
I noticed that staging has two different kustomization.yaml files:
* components/build-service/staging/base/kustomization.yaml
* components/build-service/staging/stone-stage-p01/kustomization.yaml

^ should we update the second one as well?

Reply:
Hmm, actually - wouldn't it be possible to keep the resources in `base` so that development and staging would take them from `base`, and override them only in production?

Reply:
> ^ should we update the second one as well?
`components/build-service/staging/stone-stage-p01/kustomization.yaml` is referring the `- ../base` in the resources, so both is covered.
> Hmm, actually - wouldn't it be possible to keep the resources in base so that development and staging would take them from base, and override them only in production?
Could you please explain a bit more on this point? Not able to get what is change that you are suggesting.

Reply:
I was thinking that most things could stay in `components/build-service/base/kustomization.yaml`.
In `prod/kustomization.yaml`, we would only override the ref for
```
- https://github.com/redhat-appstudio/build-service/config/default?ref=45e8fe7c3b036d1df3eee11c1d83e0633e451c07

images:
- name: quay.io/redhat-appstudio/build-service
  newName: quay.io/redhat-appstudio/build-service
  newTag: 45e8fe7c3b036d1df3eee11c1d83e0633e451c07
```
But it looks like that doesn't work, so nevermind :sweat_smile: